source/_components/cover.garadget.markdown

```diff
@@ -43,3 +43,50 @@ Configuration variables:
 
 
 If provided, the **access_token** will be used, otherwise the **username** and **password** will be used to automatically generate an access token at start time.
+
+
+**Example with more detail:**
+<p class='img'>
+<img src='/source/images/components/garadget/cover_garadget_details.png' />
```

Comment:
`git add /source/images/components/garadget/cover_garadget_details.png` then commit and push to this branch.